UI/Web/src/app/user-settings/api-key/api-key.component.ts

```diff
@@ -43,8 +44,10 @@ export class ApiKeyComponent implements OnInit, OnDestroy {
     this.onDestroy.complete();
   }
 
-  async copy() {
-    await navigator.clipboard.writeText(this.key);
+  async copy(input: any) {
```

Comment:
First, don't use any types, this should be a string. Second, why are you passing it from the template and not just referencing this.key directly?

Reply:
Reading the copy code vs how it's invoked, I'm confused. It's supposed to be an HTMLInputElement type, but not sure how the template is getting a reference to this. 

I think the mode ideal way would be to setup a ViewChild to reference the input:

@ViewChild('keyInput') inputElem: ElementRef<HTMLInputElement>;

Then in copy, use this.inputElem.nativeElement to use the .select and setSelectionRange, etc.

Reply:
Resolved in 9692ab3